src/main/java/racing/domain/UserInput.java

```diff
@@ -0,0 +1,79 @@
+package racing.domain;
+
+import java.util.ArrayList;
+import java.util.InputMismatchException;
+import java.util.List;
+import java.util.Scanner;
+import java.util.StringTokenizer;
+
+public class UserInput {
+    final private static int MIN_LENGTH = 1;
+    final private static int MAX_LENGTH = 5;
+
+    public static List<String> getCarNames() {
+        System.out.println("경주할 자동차 이름을 입력하세요(이름은 쉼표(,)를 기준으로 구분).");
+
+        Scanner sc = new Scanner(System.in);
+        String input = sc.nextLine();
+        List<String> nameList;
+
+        try {
+            nameList = splitStringByComma(input);
+            checkCarCount(nameList);
+        } catch (IllegalArgumentException e) {
+            System.out.println(e.getMessage());
+            return getCarNames();
+        }
+        return nameList;
```

Comment:
아래와 같이 변경해볼 수 있을 것 같아요 😉
```suggestion
        try {
            List<String> names = splitStringByComma(input);
            checkCarCount(names);
            return names;
        } catch (IllegalArgumentException e) {
            System.out.println(e.getMessage());
            return getCarNames();
        }
```

Reply:
오! 감사합니다 ㅎㅎ 말씀해주신대로 코드를 변경하면 가독성이 높아져서 그런게 맞을지 궁금합니다..!

Reply:
가독성 측면에서도 좋아지는 것이 맞고, 구글 코드 컨벤션에서도 변수의 `선언`과 `초기화`는 동시에 진행하라고 가이드하고있어요! 😃